### ISSUE TITLE
Why

### ISSUE THREAD

**Issue by limoka** [prodzpod/NewBepin#1]
The question is simple. Why have you done this? What's the point of reintroducing BepInEx when the game already has a inbuilt mod loader and modding framework.

**Comment by prodzpod** [comment]
have you tried sharing any sort of modpacks with friends
also ecs kind of sucks sorry lol

**Comment by limoka** [comment]
The fact that ingame mod manager sucks doesn't mean we need a whole another modding framework.
Also, ECS, in fact, does NOT suck. It powers the whole game, and you not understanding it is a you problem.

The solution to your "problem" is called manual mod installation or using Core Launcher

**Comment by prodzpod** [comment]
yeah ive read the wiki and thats why this exists now. idk why ur even interested in this lol im just providing extra options (it works w modio guys, it'd be actually splitting the community and your aggression would be understandable if it didnt)

**Comment by limoka** [comment]
Suffice to say, such options might indeed split the community. I do not think BepInEx makes sense to be used in the current situation. Distributing mods via Thunderstore for it's mod manager is not a good solution to your problem, even if it seems so.

I do agree that ingame mod manager sucks, and that devs do need to put more effort into improving it.
Also I don't know where you saw "aggression" from me. I just wanted to know why this was done.

**Comment by Smxrez** [comment]
Bepinex and r2modman is honestly just better in alot of ways
And like they said it works with modio how would it be splitting the community

**Comment by prodzpod** [comment]
yeah im not gonna distribute mods from modio, just the modpack data and the downloads will still be done via modio (see the new repo that just appeared)

**Comment by limoka** [comment]
I see, but don't you realize that Core Launcher is trying to solve the same issue? Perhaps instead of trying to jerry rig Thunderstore into collaboration with mod io and the game, you would just add mod pack support to it?

**Comment by limoka** [comment]
https://github.com/super-miner/Core-Launcher

**Comment by prodzpod** [comment]
thatd be the correct way to do it, and feel free to do so. its kind of a lot of work tho
